pbs/pbsrequest.go

```diff
@@ -237,9 +237,11 @@ func ParsePBSRequest(r *http.Request, cache cache.Cache) (*PBSRequest, error) {
 	if pbsReq.App == nil {
 		pbsReq.Cookie = ParsePBSCookieFromRequest(r)
 
-		// this would be for the shared adnxs.com domain
-		if anid, err := r.Cookie("uuid2"); err == nil {
-			pbsReq.Cookie.TrySync("adnxs", anid.Value)
+		// Host has right to leverage private cookie store for user ID
+		if uid, _, _ := pbsReq.Cookie.GetUID(hostCookieSettings.Family); uid == "" && hostCookieSettings.CookieName != "" {
```

Comment:
Can be simplified to `if pbsReq.Cookie.HasLiveSync(hostCookieSettings.Family) && hostCookieSettings.CookieName != ""`